packages/compass-aggregations/src/components/stage-preview/stage-preview.jsx

```diff
@@ -135,21 +142,27 @@ class StagePreview extends Component {
       }
       return (<div className={styles['stage-preview-out']} />);
     }
+
+    const previewOutText =
+      process.env.COMPASS_SHOW_NEW_AGGREGATION_TOOLBAR !== 'true'
+        ? 'The $out operator will cause the pipeline to persist the results to the specified location (collection, S3, or Atlas). If the collection exists it will be replaced. Please confirm to execute.'
+        : 'The $out operator will cause the pipeline to persist the results to the specified location (collection, S3, or Atlas). If the collection exists it will be replaced.'
+
     return (
       <div className={styles['stage-preview-out']}>
         <div className={styles['stage-preview-out-text']}>
-          The $out operator will cause the pipeline to persist the results
-          to the specified location (collection, S3, or Atlas). If the collection exists it will be
-          replaced. Please confirm to execute.
-        </div>
-        <div className={styles['stage-preview-out-button']}>
-          <TextButton
-            dataTestId="save-out-documents"
-            text="Save Documents"
-            className="btn btn-xs btn-primary"
-            clickHandler={this.onSaveDocuments}
-          />
+          {previewOutText}
```

Comment:
Should with wrap this in a `Body` element? I'm thinking maybe we can add some padding there to.
<img width="861" alt="Screen Shot 2022-05-08 at 4 48 45 PM" src="https://user-images.githubusercontent.com/1791149/167315318-dbaec8a4-80e8-4b5f-b88d-236b32afbf84.png">

Reply:
Done in [bd876db](https://github.com/mongodb-js/compass/pull/3060/commits/bd876dbd5a119c74b3b2251bef65b69a21b7238d)